api/v1alpha1/openapi.yaml

```diff
@@ -3131,12 +3133,55 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Status'
+  /api/v1/organizations:
+    get:
+      tags:
+        - organization
+      summary: List organizations
+      description: Retrieves a list of organizations.
```

Comment:
I'm assuming that the organizations that someone can retrieve would also be only those they have access to? Is it necessary to clarify in the description?